test/weet.test.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable no-unused-vars */
+const request = require('supertest');
+const axios = require('axios');
+const app = require('../app');
+const { regularUser, signinRegularCredentials } = require('./mocks/weets');
+const {
+  createWeetInput,
+  createWeetOutput,
+  createLongWeetInput,
+  createLongWeetOutput
+} = require('./mocks/weets');
+
+jest.mock('axios');
+describe.only('Weets', () => {
+  /* eslint-disable init-declarations */
+  let signedInUser;
+  let regularToken;
+  let response = {};
+  let responseLong = {};
+  beforeAll(async () => {
+    await request(app)
+      .post('/users')
+      .send(regularUser);
+
+    signedInUser = await request(app)
+      .post('/users/sessions')
+      .send(signinRegularCredentials);
+    regularToken = signedInUser.body.data.token;
+  });
+
+  describe('Weet should be created', () => {
+    beforeAll(async () => {
+      axios.get.mockImplementationOnce(() => Promise.resolve(createWeetInput));
+      response = await request(app)
+        .post('/weets')
+        .set('Authorization', `Bearer ${regularToken}`);
+      axios.get.mockImplementationOnce(() => Promise.resolve(createLongWeetInput));
+      responseLong = await request(app)
+        .post('/weets')
+        .set('Authorization', `Bearer ${regularToken}`);
```

Comment:
You could add all this process in the describe for the Weet with a length of 140.